### DISCUSSION TITLE
How to count serverless function in Express.Js deployment

### DISCUSSION THREAD

**Discussion by flitzcore** [vercel/community#6186] [Help]
### Summary

So I want to deploy this Express template: https://github.com/hagopj13/node-express-boilerplate.git
I made some changes based on this article: https://vercel.com/guides/using-express-with-vercel
everything run smoothly till I got:
```
Error: No more than 12 Serverless Functions can be added to a Deployment on the Hobby plan. Create a team (Pro plan) to deploy more. Learn More: https://vercel.link/function-count-limit
```

How to know how many serverless function in my program? Does it count the number of my routes? Does it count every file there even if it does nothing?

### Steps to Reproduce

```
git clone --depth 1 https://github.com/hagopj13/node-express-boilerplate.git
cd node-express-boilerplate
npx rimraf ./.git
```
rename src as api
add vercel.json

vercel.json:
```
{
  "rewrites": [
    { "source": "/(.*)", "destination": "/api" }
  ]
}
```

**Comment by amyegan** [comment]
Hey @flitzcore. Each file in the `/api` directory would add to the count. Utility files should start with either `_` or `.` as a prefix. Files with the prefix will not be turned into serverless functions.

https://vercel.com/docs/functions/concepts#adding-utility-files-to-the-/api-directory
